lib/routes.js

```diff
@@ -72,7 +86,10 @@ export default function routes(req, res, logger) {
     log.end().addDefaultFields(clientInfo);
 
     if (req.url === '/_/healthcheck') {
-        return setHealthCheckResponse(req, res, log);
+        if (!checkIP(clientInfo.clientIP)) {
+            return writeResponse(res, errors.Forbidden, log);
```

Comment:
will you please change this to errors.AccessDenied to be consistent with what Rahul did in the other projects and because the message that goes along with forbidden ('authentication failed') doesn't quite make sense here.  See https://github.com/scality/Arsenal/blob/master/errors/arsenalErrors.json#L346